x-pack/plugins/siem/public/alerts/components/signals/actions.tsx

```diff
@@ -65,8 +68,19 @@ export const updateSignalStatusAction = async ({
     await updateSignalStatus({ query: queryObject, status });
     // TODO: Only delete those that were successfully updated from updatedRules
     setEventsDeleted({ eventIds: signalIds, isDeleted: true });
+
+    const successTitle =
+      status === 'closed'
+        ? i18n.CLOSED_ALERT_SUCCESS_TOAST(signalIds.length)
+        : i18n.OPENED_ALERT_SUCCESS_TOAST(signalIds.length);
+
+    displaySuccessToast(successTitle, dispatchToaster);
   } catch (e) {
-    // TODO: Show error toasts
+    const errorTitle =
+      status === 'closed'
+        ? i18n.CLOSED_ALERT_FAILED_TOAST(signalIds.length)
+        : i18n.OPENED_ALERT_FAILED_TOAST(signalIds.length);
+    displayErrorToast(errorTitle, [e.message], dispatchToaster);
```

Comment:
When performing bulk actions on _all_ signals via the `Select all ### signals` feature, the tooltip will only display the max signals per page, e.g.

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/83453230-5a098d80-a417-11ea-8226-0562e60c2340.gif" />
</p>

While you could get the `totalSelected` from the parent `SignalsTable` component, it'd be best to grab this from the `response` as not all signals could've been updated successfully. Here's the `response`:

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/83454633-b53c7f80-a419-11ea-9381-17e9d615119e.png" />
</p>

That said, looks like we've got a bug here:

https://github.com/elastic/kibana/blob/535fe89ae8f7378af1ee51899ac997feecca07e9/x-pack/plugins/siem/public/alerts/components/signals/index.tsx#L189

as we're not including the selected status as a query constraint, so it'll end up updating all signals for the given `ruleId`/`datetimeRange`. So for example if there's `25 Open` & `275 Closed`, selecting all closed will return `300`.

To prevent a yak-shaving-adventure, I recommend adding support for reading `total`/`updated` from the `response`, and then either opening an issue for the query bug, or following up with a fix.